src/shared.ts

```diff
@@ -21,6 +21,8 @@ export type ConnectElementHTMLName =
   | "stripe-connect-instant-payouts"
   | "stripe-connect-issuing-card"
   | "stripe-connect-issuing-cards-list"
+  | "stripe-connect-financial-account"
```

Comment:
You are missing the setters!

https://github.com/stripe/connect-js/blob/1dd5e7fbf73211c5f8bd03544e1f177e00c8ae15/types/config.ts#L22